twitter/Models/User.h

```diff
@@ -0,0 +1,26 @@
+//
+//  User.h
+//  twitter
+//
+//  Created by Iris Fu on 6/20/22.
+//  Copyright © 2022 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface User : NSObject
+
+// MARK: Properties
+
+@property (nonatomic, strong) NSString *name;
```

Comment:
It's usually a good idea to declare `NSString *` properties as `copy` instead of `strong` to promote immutability. If the string is not copied, someone could assign it to an instance of `NSMutableString *` and later modify that mutable string, causing your property to also get modified (unexpectedly).